build-ci-wheels/action.yml

```
@@ -0,0 +1,82 @@
+name: "Build wheels for a C-extension Python library"
+description: "Build wheels for a C-extension Python library"
+
+inputs:
+  cpython-version:
+    description: 'Desired CPython version'
```

Could we name this `python-version` as we are doing in other actions?

```suggestion
  python-version:
    description: 'Desired Python version'
```

---

Remember to change it in other parts of this file.